src/LogEntry.php

```diff
@@ -95,7 +95,7 @@ public function __toString()
     private function replaceCurlyBrackets($logMessage, $logContext)
     {
         return preg_replace_callback(
-            '(\{(.*?)\})',
+            '(\{(.*?)})',
```

Comment:
This is probably an error? The opening brace is escaped, the closing one should be too?

Reply:
PhpStorm inspection shows second escape to be superfluous.

Reply:
Looks like that’s the case (in regex101) .. still not convinced that it helps to understand the regex.